std/random.d

```diff
@@ -415,8 +415,8 @@ $(D x0).
         return _x;
     }
 
-///
-    @property typeof(this) save() @safe pure nothrow
```

Comment:
Ok, I'll add them back. I think I will make a pass over the whole module to add all possible attributes.